Add content from community doc PRs (640, 782, 785) (#73)
rancher/harvester-product-docs: versions/v1.4/modules/en/pages/networking/cluster-network.adoc

The xref:../hosts/witness-node.adoc[witness node] is generally not involved in the custom cluster network.
====

== Configuration

=== Create a new cluster network

[TIP]
====

> To simplify cluster maintenance, create one network configuration for each node or group of nodes. Without dedicated network configurations, certain maintenance tasks (for example, replacing old NICs with NICs in different slots) will require you to stop and/or migrate the affected virtual machines before updating the network configuration.

Reviewer: ```suggestion
To simplify cluster maintenance, create one network configuration for each node or group of nodes. Otherwise, certain maintenance tasks, such as replacing old NICs with NICs in different slots, will require you to stop or migrate the affected virtual machines, or both, before updating the network configuration.
```

Author: @sushant-suse The second sentence in your draft is disjointed because of the many commas. Also, "otherwise" isn't a direct replacement for "without dedicated network configurations". I'll keep the original wording for now.

Reviewer: Thanks for the feedback, it makes sense. I see your point about "otherwise" being too general and the second sentence getting a bit heavy with commas. I'm good with keeping the original wording as is.
